phys2bids/io.py

```diff
@@ -1,18 +1,21 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
-"""phys2bids interface for txt files."""
-
+"""phys2bids interface for acqknowledge files."""
```

Comment:
Why do you say it's the interface for acqknowledge files? We read txt files too.

Reply:
resolved